concepticondata/conceptlists/Steinthal-1875-1565.tsv

```diff
@@ -0,0 +1,1566 @@
+Steinthal-1875-1565-10	10	Die Sonne ist aufgegangen, untergegangen, brennt heiss.	Die Sonne ist untergegangen.	The sun has set.		
+Steinthal-1875-1565-11	11	Die Sonne ist aufgegangen, untergegangen, brennt heiss.	Die Sonne brennt heiss.	The sun burns hot.		
+Steinthal-1875-1565-12	12	Mond	Mond	moon	1313	MOON
+Steinthal-1875-1565-13	13	Neumond	Neumond	new moon	3691	NEW MOON
+Steinthal-1875-1565-14	14	Es ist Mondfinsterniss.	Es ist Mondfinsterniss.	A lunar eclipse occurs.	2285	ECLIPSE
+Steinthal-1875-1565-15	15	Sterne	Sterne	stars	1430	STAR
+Steinthal-1875-1565-16	16	Komet	Komet	comet		
+Steinthal-1875-1565-17	17	Sternschnuppe	Sternschnuppe	shooting star	2835	SHOOTING STAR
+Steinthal-1875-1565-18	18	Nebel, er steigt, fällt.	Nebel	fog	249	FOG
+Steinthal-1875-1565-19	19	Nebel, er steigt, fällt.	Nebel steigt.	Fog rises.		
+Steinthal-1875-1565-20	20	Nebel, er steigt, fällt.	Nebel fällt.	Fog falls.		
+Steinthal-1875-1565-21	21	Regen (vielleicht verschieden benannt nach der Jahreszeit, oder der Heftigkeit)	Regen	rain	658	RAIN (PRECIPITATION)
+Steinthal-1875-1565-22	22	Es regnet.	Es regnet.	It rains.	1253	RAIN (RAINING)
+Steinthal-1875-1565-23	23	Gewitter	Gewitter	thunderstorm		
+Steinthal-1875-1565-24	24	Donner	Donner	thunder	1150	THUNDER
+Steinthal-1875-1565-25	25	Es donnert.	Es donnert.	to thunder	2378	THUNDER (VERB)
+Steinthal-1875-1565-26	26	Blitz	Blitz	lightning	1654	LIGHTNING
+Steinthal-1875-1565-27	27	Es blitzt.	Es blitzt.	Lightning strikes.		
+Steinthal-1875-1565-28	28	Es kommt ein Gewitter.	Es kommt ein Gewitter.	A thunderstorm is coming.		
+Steinthal-1875-1565-29	29	Der Regen hört auf.	Der Regen hört auf.	The rain stops.		
+Steinthal-1875-1565-30	30	Regenbogen	Regenbogen	rainbow	1733	RAINBOW
+Steinthal-1875-1565-31	31	Thau	Thau	dew	1977	DEW
+Steinthal-1875-1565-32	32	Schnee	Schnee	snow	784	SNOW
+Steinthal-1875-1565-33	33	Hagel	Hagel	hail	609	HAIL
+Steinthal-1875-1565-34	34	Reif	Reif	hoar frost	2942	HOARFROST
+Steinthal-1875-1565-35	35	Eis schmilzt, thaut.	Eis schmilzt.	Ice melts.		
+Steinthal-1875-1565-36	36	Eis schmilzt, thaut.	Eis thaut.	Ice thaws.		
+Steinthal-1875-1565-37	37	Tag, bricht an, geht zu Ende	Tag	day	1225	DAY (NOT NIGHT)
+Steinthal-1875-1565-38	38	Tag, bricht an, geht zu Ende	Tag bricht an.	daybreak		
+Steinthal-1875-1565-39	39	Tag, bricht an, geht zu Ende	Tag geht zu Ende.	Day comes to an end.		
+Steinthal-1875-1565-40	40	Nacht, sie kommt herein	Nacht	night	1233	NIGHT
+Steinthal-1875-1565-41	41	Nacht, sie kommt herein	Nacht kommt herein.	Night falls.		
+Steinthal-1875-1565-42	42	Morgen	Morgen	morning	1339	MORNING
+Steinthal-1875-1565-43	43	Mittag	Mittag	noon	12	MIDDAY
+Steinthal-1875-1565-44	44	Abend	Abend	evening	1629	EVENING
+Steinthal-1875-1565-45	45	Morgenroth	Morgenroth	dawn	729	DAWN
+Steinthal-1875-1565-46	46	Abendröthe	Abendröthe	dusk	2297	DUSK
+Steinthal-1875-1565-47	47	Es ist finster.	Es ist finster.	It is gloomy.	1552	GLOOMY
```

Comment:
Our 'GLOOMY' refers to the emotional state (sadness), not the description of light.

Reply:
I missed that. Will unmap.